subprojects/core/src/main/java/org/gradle/api/internal/project/taskfactory/AbstractOutputPropertyAnnotationHandler.java

```diff
@@ -40,7 +38,6 @@ public void validate(String propertyName, Object value, Collection<String> messa
             public void update(TaskInternal task, final Callable<Object> futureValue) {
                 createPropertyBuilder(context, task, futureValue)
                     .withPropertyName(context.getName())
-                    .withPathSensitivity(getPathSensitivity(context))
```

Comment:
We could. Would that mean that we then would remove `TaskFilePropertyBuilder.withPathSensitivity` and push this method down to `TaskInputFilePropertyBuilder`? Is `TaskFilePropertyBuilder` a return value from the public API?